app/webpacker/controllers/toggle_controller.js

```diff
@@ -4,9 +4,22 @@ export default class extends Controller {
   static targets = ["content"];
 
   toggle(event) {
+    event.stopImmediatePropagation()
```

Comment:
Was `event.stopImmediatePropagation()` required in order to fix a specific problem? Or did you add it reflexively? :sweat_smile:

Reply:
Hmm interesting. If I comment out this line the tests fail. But... if I test it manually in the browser there's no problem at all and it works as expected in both cases where it's used. :thinking:

Reply:
Yeah this is a bit mysterious, I put it in because of that failing test alright but maybe it's not a good idea. I have now skipped that test with a comment. I wonder if it's something to do with how Jest works because it passes if I comment out other tests in that file. If there are other tests in the file it fails and if you add a `console.log` to output something in the controller, it appears as though the element is being clicked more times than it should be, like event listeners are being added multiple times.